hoomd/hpmc/pytest/test_remove_drift.py

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) 2009-2021 The Regents of the University of Michigan
```

Comment:
Every test is restricted to serial. Does the `UpdaterRemoveDrift` not work in MPI simulations?

Reply:
This is not resolved, one test still is marked `@skip_mpi`.